test/test_suite.py

```diff
@@ -198,3 +213,191 @@ class TableDDLTest(_TableDDLTest):
     )
     def test_underscore_names(self):
         pass
+
+
+class DateTest(_DateTest):
+    def test_round_trip(self):
+        """
+        SPANNER OVERRIDE:
+
+        Cloud Spanner supports tables with empty primary key, but
+        only single one row can be inserted into such a table -
+        following insertions will fail with `400 id must not be NULL in table date_table`.
+        Overriding the tests and add a manual primary key value to avoid the same failures.
+        """
```

Comment:
Minor nit:
```
        Cloud Spanner supports tables with an empty primary key, but
        only one row can be inserted into such a table -
        following insertions will fail with `400 id must not be NULL in table date_table`.
        Overriding the tests to add a manual primary key value to avoid the same failures.
```